commands/search.js

```diff
@@ -61,6 +62,11 @@ class Search extends Command {
   }
 
   async run(message, args, level) { // eslint-disable-line no-unused-vars
+    if (this.activeSearches.has(message.author.id))
+      return;
```

Comment:
Couldn't this be turned into a one liner with `if (this.activeSearches.has(message.author.id)) return;`?

Reply:
It could, but I like the clean look of using an extra, indented line.

Reply:
Fair enough